packages/website/pages/api/user/[...tokens].js

```diff
@@ -0,0 +1,71 @@
+const secret = `${Number.MAX_SAFE_INTEGER}.${Number.MAX_SAFE_INTEGER * 2}.${Number.MAX_SAFE_INTEGER * 3}.${
```

Comment:
surprising filename `[...tokens].js` !?

Reply:
oh my it is just how next.js api dynamic routes work https://nextjs.org/docs/api-routes/dynamic-api-routes

Reply:
:grandad:

Reply:
Yeah! It doesn't actually export in the static site or anything, but this was a nice mock API to easily test some weird values with when running locally.